src/woo_publications/publications/api/serializer.py

```diff
@@ -29,14 +33,34 @@ class Meta:  # type: ignore
         )
 
 
+class EigenaarSerializer(serializers.Serializer):
+    display_name = serializers.CharField(
+        read_only=True,
+        help_text=_(
+            "obtained from the audit trail request headers, extracted from the audit trails/log entries."
+        ),
+    )
+    identifier = serializers.CharField(
+        read_only=True,
+        help_text=_(
+            "arbitrary, unique-ish string. Could be a `sub` claim from OIDC, but for our perspective this data has no 'meaning'."
+        ),
+    )
+
+
 class PublicationSerializer(serializers.ModelSerializer):
+    eigenaar = serializers.SerializerMethodField(
+        help_text=_("De eigenaar van de publicatie."), method_name="get_acting_user"
+    )
```

Comment:
`SerializerMethodField` is something I hope you forget that it exists - it's really a last resort to get the data out.

The reason is that it doesn't play nice with the generated API documentation - evidenced by your need for `extend_schema_field`, and that is a risk of implementation and documentation diverging.

Ultimately serialization is about "I have rich python/django objects, translate that into something that I can chuck over the wire and put in JSON". Serializers themselves should really be as boring as possible.

So, what is the correct approach then?

1. Move the lookups/data retrieval into their appropriate layer - either the view or the model layer. In this case, the model layer is easiest.
2. Define the serializer properly "inline" and mark it as read-only

```suggestion
    eigenaar = EigenaarSerializer(
        source="get_owner",
        label=_("owner"),
        help_text=_("The creator of the publication, derived from the audit logs."),
        allow_null=True,
        read_only=True,
    )
```

Notable changes:

- I made everything English
- I use the `EigenaarSerializer` directly - it knows how to serialize the `ActingUser` data
- I added `allow_null=True` - rather than returning an empty object if we don't have the information, let's just return `null` for that, as that holds the right semantic value. The client then knows if it's not None, that it will  be an object with the keys `displayName` and `identifier`
- I specified `source="get_owner"`, which is a method on the model.

`PublicationSerializer` gets a `Publication` instance, so when it tries to serialize the `eigenaar`, it will look up the `get_owner` attribute, see that it's a method that can be called, and call it. That returns the dictionary with information, or `None` - if it's the latter, it won't even attempt to use `EigenaarSerializer`, but will stick with `None`. If we have the data, the return value of `get_object` is passed to `EigenaarSerializer` and we get the nested object.

The `get_owner` implementation that works is pretty much what you have in `get_acting_user`:

```py
    def get_owner(self) -> ActingUser | None:
        """
        Extract the owner from the audit trails.
        """
        qs = TimelineLogProxy.objects.for_object(self)  # type: ignore reportAttributeAccessIssue
        try:
            log = qs.get(extra_data__event=Events.create)
        except TimelineLogProxy.DoesNotExist:
            return None
        assert isinstance(log, TimelineLogProxy)
        return log.acting_user[0]
```
with some proper type hints/assertions - I'll have to take a look at the library timeline-logger library :)